experimental/dds/tree2/src/feature-libraries/sequence-field/sequenceFieldToDelta.ts

```diff
@@ -45,21 +47,30 @@ export function sequenceFieldToDelta<TNodeChange>(
 		} else if (areInputCellsEmpty(mark) && areOutputCellsEmpty(mark)) {
 			// The cell starting and ending empty means the cell content has not changed,
 			// unless transient content was inserted/attached.
-			if (markIsTransient(mark)) {
-				const oldId = nodeIdFromChangeAtom(mark.cellId);
-				if (!areEqualChangeAtomIds(mark.cellId, mark.transientDetach)) {
-					// TODO: handle transient move-in/return-to
-					assert(isInsert(mark), "Unexpected non-insert transient mark");
-					if (mark.content !== undefined) {
+			if (isTransientEffect(mark)) {
+				if (isMoveDestination(mark.attach) && isMoveSource(mark.detach)) {
+					assert(mark.changes === undefined, "Transient moves should not have changes");
+					continue;
+				}
+
+				const outputId = getOutputCellId(mark, revision, undefined);
```

Comment:
It looks like if mark.detach is a move source with a `finalEndpoint`, then we're not using it, which surprises me.

Reply:
Same question for MoveOut & ReturnFrom below

Reply:
The scheme used here is to detach the nodes with the move source's ID, and then reattach them using that same ID. So the destination's ID is never used in the delta.

Reply:
Ah, right, there's an asymmetry right now because we have to pick one ID.
A comment would be nice.